jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/java/JavaHandlerCompletion.kt

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.java
+
+import com.intellij.codeInsight.completion.PrefixMatcher
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.openapi.project.Project
+import software.aws.toolkits.jetbrains.services.lambda.completion.HandlerCompletion
+
+class JavaHandlerCompletion : HandlerCompletion {
+
+    override fun isSupported(): Boolean = false
```

Comment:
Is there any point having these? Can't we just not implement them for the JAVA / PYTHON run times - and their 'absence' determines whether or not they're supported?

Reply:
Updated.